labelbox/data/serialization/ndjson/classification.py

```diff
@@ -85,12 +100,14 @@ class NDRadioSubclass(NDFeature):
 
     def to_common(self) -> Radio:
         return Radio(answer=ClassificationAnswer(
-            feature_schema_id=self.answer.schema_id))
+            name=self.answer.name, feature_schema_id=self.answer.schema_id))
 
     @classmethod
-    def from_common(cls, radio: Radio,
+    def from_common(cls, radio: Radio, name: str,
                     feature_schema_id: Cuid) -> "NDRadioSubclass":
-        return cls(answer=NDFeature(schema_id=radio.answer.feature_schema_id),
+        return cls(answer=NDFeature(name=radio.answer.name,
+                                    schema_id=radio.answer.feature_schema_id),
```

Comment:
Unnecessary newline? unless github messed up formatting?

Reply:
formatter did this 🤷